libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -20,6 +20,11 @@
  * Solidity compiler.
  */
 
+#include "libyul/AsmAnalysisInfo.h"
```

Comment:
```suggestion
#include <libyul/AsmAnalysisInfo.h>
```

Can you add a style-checker rule that searches for `include "`?

Reply:
:rofl: yeah. I already looked for an option in my auto-completer to fix that, couldn't find one.